resources/scss/views/ItemList/Components/Filter/item-filter-list.scss

```diff
@@ -0,0 +1,18 @@
+.findologic-filter-wrapper #filterCollapse {
+    position: relative;
+    z-index: 0;
+}
+
+.list-controls #filterCollapse .page-content .card .form-check {
```

Comment:
Was there a specific reason to add these? The hover effect looks broken when we apply these style changes.

Reply:
I have added these, because filter badges were broken. But found a cleaner solution that doesn't break hover effect.